controller/ApplicationsController.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require("mongoose");
+const CustomError = require("../utils/customError");
+const responseHandler = require("../utils/responseHandler");
+// Application model
+const Applications = require("../models/applications");
+const Organizations = require("../models/organizations");
+
+exports.getAllApplications = async (req, res, next) => {
+  try {
+    const { orgId } = req.token;
+    if (!mongoose.Types.ObjectId.isValid(orgId)) {
+      return next(new CustomError(400, "Invalid OrganizationID"));
+    }
+
+    const organization = await Organizations.find({ _id: orgId });
+    if (!organization) {
+      return next(new CustomError(400, "Invalid organization"));
+    }
+
+    const applications = await Applications.find({ organization });
+    const allApplication = applications.map((app) => {
+      return {
+        applicationId: app._id,
+        name: app.name,
+      };
+    });
+
+    const data = [allApplication];
```

Comment:
No need to wrap your response in an array, allApplication is already an array. So u can just send allApplication or assign it direct to data (which in my opinion is not necessary) and then send it